parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -13,6 +13,7 @@ RUNC_DOWNLOADS_DIR="/opt/runc/downloads"
 K8S_DOWNLOADS_DIR="/opt/kubernetes/downloads"
 K8S_PRIVATE_PACKAGES_CACHE_DIR="/opt/kubernetes/downloads/private-packages"
 UBUNTU_RELEASE=$(lsb_release -r -s)
+OS=$(sort -r /etc/*-release | gawk 'match($0, /^(ID_LIKE=(coreos)|ID=(.*))$/, a) { print toupper(a[2] a[3]); exit }')
```

Comment:
Is this command returning exactly "MARINER" for mariner and "AZURELINUX" for azure linux (case sensitive)?
If so, could you please also add a comment for better clarification?

Reply:
Yeah, this command returns "MARINER" for mariner and "AZURELINUX" for azure linux. Will add a comment for this.

**3.0:**
![0bf16cc7-5314-4fdd-a06f-2085380668ee](https://github.com/user-attachments/assets/12322ef1-3ef3-481a-9746-bf0c5b0cb658)

**2.0:**
![8953e760-0153-4dd4-b6fd-2bb09f5ff2f4](https://github.com/user-attachments/assets/c0a94344-80c3-41ae-8af8-d6f63e4cb5c0)

Reply:
Added a comment - [commit](https://github.com/Azure/AgentBaker/pull/4808/commits/5506a9db15e2d2a56620b7b56acc13ba7c6bb9d3)